data-pipeline/src/pipeline/input_schemas.py

```diff
@@ -321,11 +300,13 @@
     "BNCH21606 (Agency supply teaching staff)": "float",
     "BNCH21403 (Energy)": "float",
     "BNCH21402 (Water and sewerage)": "float",
+    "Valid To": "string",
 }
 
 aar_central_services_index_col = "Lead_UPIN"
 aar_central_services = {
     "Lead_UPIN": "Int64",
+    "Company_Number": "string",
```

Comment:
Just checking that this column is likely to be in the same format as in the `academy_master_list` and/or the potential leading `0` is dealt with somewhere else

Reply:
Good shout: the `aar.csv` and `aar_cs.csv` files use their respective `Company_Number` to join and they're both in the same format for that purpose.

The `map_company_number()` call which zero-pads them _after_ that is retained (down on line 997).